editor/tree-sitter/check.sh

```
@@ -0,0 +1,133 @@
+  exit 1
+fi
+
+echo "==> checking ${#test_files[@]} files"
+
+fail_paths=()
+warn_paths=()
+total_pass=0
+total_skip=0
+
+for f in "${test_files[@]}"; do
+  rel="${f#$REPO_ROOT/}"
```

Shellcheck (0.11.0)

[info] Expansions inside ${..} need to be quoted separately, otherwise they match as patterns.

(SC2295)